### ISSUE TITLE
Can't import the named export 'Component' from non EcmaScript module (only default export is available)

### ISSUE THREAD

**Issue by brunolemos** [staylor/react-helmet-async#46]
Got this after upgrading to 1.x. Using create-react-app.

```
node_modules/react-helmet-async/lib/index.mjs
Can't import the named export 'Component' from non EcmaScript module (only default export is available)
```

**Comment by elskwid** [comment]
I'm seeing the same thing. `react-helmet-async 1.0.0`, `create-react-app`.

```javascript
import { HelmetProvider } from 'react-helmet-async'
```

**Comment by vxna** [comment]
fix: https://github.com/developit/preact/pull/1425

**Comment by tiarebalbi** [comment]
+1 having the same issue here.

**Comment by lumannnn** [comment]
@vxna @tiarebalbi 

If you're using webpack, adding the following rule might help. It did in our case so far:

```js
    module: {
      rules: [
        {
          type: 'javascript/auto',
          test: /\.mjs$/,
        },
        ...
     ]
  }
```

**Comment by staylor** [comment]
you can also `resolve.mainFields = ['main']` if you don't want to read `'module'`, which is the new `.mjs`

**Comment by staylor** [comment]
just published 1.0.2 https://www.npmjs.com/package/react-helmet-async

**Comment by elskwid** [comment]
@staylor thank you very much! It's working for me here. 😍 🙇
